spec/services/anno_auth_validator_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+require 'webmock/rspec'
+
+RSpec.describe AnnoAuthValidator do
+
+  context 'when authorize is called' do
+    describe 'validate canvas and user' do
+      before(:each) do
+        Rails.application.secrets.jwt_password = 'abc123'
+        Rails.application.config.JWT_Canvas_Verification_Url = 'http://testserver.com'
+        allow(JWT).to receive(:decode).and_return([{"group_id" => "10", "user_id" => "100"}])
+      end
+
+      it 'check valid JWT' do
+        stubAnnoAuth("10", "true")
+        expect(described_class.authorize("Bearer someencryptedtoken", "http://canvas.testserver.com/canvas/12345")).to eq(true)
```

Comment:
You can use the matcher `be_truthy` instead of explicitly checking for a return value of `true` unless that's important; in this case since all we're doing is passing the outcome to an `if` statement `be_truthy` is ultimately what we care about.

Also, on a line this long I might extract the `described_class.authorize(...)` piece to a variable (something like `results`) so that everything fits in 80 characters; you'd then just test `expect(results).to be_truthy`.